userspace/otpilot/src/main.rs

```diff
@@ -195,11 +188,11 @@ fn run() -> TockResult<()> {
                 Ok(()) => {}
                 Err(why) => {
                     // Ignore error from writeln. There's nothing we can do here anyway.
-                    let _ = writeln!(console, "Device: Error processing SPI packet: {:?}", why);
+                    let _ = writeln!(console, "SPI processor: Error {:?}", why);
                     if spi_device::get().is_busy_set() {
                         if let Err(_) = spi_device::get().end_transaction_with_status(true, false) {
                             // Ignore error from writeln. There's nothing we can do here anyway.
-                            let _ = writeln!(console, "Device: Error ending transaction.");
+                            let _ = writeln!(console, "SPI device: end_transaction error.");
```

Comment:
Why not print the error?

(Guessing it is content free, except insofar as it informs us that something didn't go right.)

Reply:
We don't have a printer for TockResult right now. This is covered under #227 .

Reply:
:ack: